src/Numerics/Mesh/Interpolation.jl

```diff
@@ -225,9 +224,9 @@ struct InterpolationBrick{
 
         end # el loop
 
-        m_ξ1, _ = GaussQuadrature.legendre(FT, qm[1], GaussQuadrature.both)
-        m_ξ2, _ = GaussQuadrature.legendre(FT, qm[2], GaussQuadrature.both)
-        m_ξ3, _ = GaussQuadrature.legendre(FT, qm[3], GaussQuadrature.both)
+        m_ξ1, _ = Elements.lglpoints(FT, qm[1]-1)
+        m_ξ2, _ = Elements.lglpoints(FT, qm[2]-1)
+        m_ξ3, _ = Elements.lglpoints(FT, qm[3]-1)
         wb1 = Elements.baryweights(m_ξ1)
         wb2 = Elements.baryweights(m_ξ2)
         wb3 = Elements.baryweights(m_ξ3)
```

Comment:
do you need to update baryweights?

Reply:
I looked at this. It should work in its current form.